repository/mock_repo.go

```diff
@@ -82,6 +82,17 @@ func (r *mockRepoForTest) RmConfigs(keyPrefix string) error {
 	return nil
 }
 
+func (r *mockRepoForTest) ListRemotes() (map[string]string, error) {
+	remotesMap := make(map[string]string)
+
+	for key, value := range r.config {
+		if strings.HasPrefix(key, "remote.origin.url") {
```

Comment:
The remote could be different than `origin`. As an example, here is what I have in my config:

```
git config --list | grep "remote\."
remote.origin.url=git@github.com:MichaelMure/git-bug.git
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
remote.JonLundy.url=git@github.com:JonLundy/git-bug.git
remote.JonLundy.fetch=+refs/heads/*:refs/remotes/JonLundy/*
remote.vandesign.url=git@github.com:vandesign/git-bug.git
remote.vandesign.fetch=+refs/heads/*:refs/remotes/vandesign/*
remote.adamslc.url=git@github.com:adamslc/git-bug.git
remote.adamslc.fetch=+refs/heads/*:refs/remotes/adamslc/*
remote.steap.url=git@github.com:Steap/git-bug.git
remote.steap.fetch=+refs/heads/*:refs/remotes/Steap/*
remote.programfan.url=git@github.com:ProgramFan/git-bug.git
remote.programfan.fetch=+refs/heads/*:refs/remotes/programfan/*
remote.seeduvax.url=git@github.com:seeduvax/git-bug.git
remote.seeduvax.fetch=+refs/heads/*:refs/remotes/seeduvax/*
remote.sladyn98.url=git@github.com:sladyn98/git-bug.git
remote.sladyn98.fetch=+refs/heads/*:refs/remotes/sladyn98/*
```

Reply:
So what do you want the tests to check for? Do you want me to create a custom array of origins or names and test for each?

Reply:
You should be able to do:

`mockRepo.StoreConfig("remote.origin.url", "git@github.com:MichaelMure/git-bug.git")`
`mockRepo.ListRemotes()` // return a map with `origin` --> `git@github.com:MichaelMure/git-bug.git`